### DISCUSSION TITLE
Dark theme support

### DISCUSSION THREAD

**Discussion by imsorrywhat** [chadwpalm/PrerollPlus#5] [Ideas]
Would be great to have a dark theme option.

**Comment by chadwpalm** [comment]
Check new tag chadwpalm/prerollplus:fix-inline-css-0.1.5.21

New icon on top right next to menu to toggle dark and light modes. Let me know if it is to your liking. It took me a few days to do because I took the time to rewrite the code to move all inline CSS to external files while adding in the support for dark mode. It's something I needed to do anyway and will make future styling changes easier.

**Reply by imsorrywhat** [reply]
Yup, works great! Love how fast the change happens. Keep forgetting to ask...is the red vs orange theme intentional?

**Reply by chadwpalm** [reply]
Red vs orange theme?

**Reply by imsorrywhat** [reply]
yeah, the red line across the banner and in the icon? was expecting orange to go along with plex colo scheme.

**Reply by chadwpalm** [reply]
Ohhh gotcha. When I made my app Lumunarr, I wanted it to be like Sonarr and Radarr which are blue and yellow respectively, so I made it a purple color. Then when I made PR+ I wanted something different so I went with red.

Some backstory.....my app Lumunarr was originally called HuePlex, but Plex contacted me and asked me to change the name cause it couldn't have Plex in it. That's another reason I don't want to mimic Plex's color schemes.

**Reply by imsorrywhat** [reply]
oh okay that makes sense!

**Comment by imsorrywhat** [comment]
Will this be merged to the develop branch? Had to rebuild the docker container and was wondering where the Dark Mode went.

Also, when I use this tag, there's a persistent alert that an update is available.

**Reply by chadwpalm** [reply]
I wanted to lump these changes with the ones I was doing for the file monitoring. I'll try to get this merged today.

You're seeing the persistent update alert because the version is different than what is currently in the dev branch. That'll get rectified when I do the merge.

**Reply by imsorrywhat** [reply]
oh okay. no rush, was just wondering. :)

**Reply by chadwpalm** [reply]
Just FYI, I've rolled the new changes (including dark mode) into the develop branch.
